build.gradle

```diff
@@ -11,14 +11,14 @@ buildscript {
     }
     dependencies {
         classpath "org.grails:grails-gradle-plugin:$grailsVersion"
-        classpath "org.grails.plugins:hibernate5:${gormVersion - '.RELEASE'}"
+        classpath "org.grails.plugins:hibernate5:7.0.0"
```

Comment:
better use `${gormVersion}` instead of 7.0.0, you already have `gormVersion=7.0.2.RELEASE` in the gradle.properties

Reply:
@zhuravskiy the hibernate plugin isn't released in lock-step w/ the gorm plugin. The latest version available of `org.grails.plugins:hibernate5` is 7.0.1 (see [here](https://repo.grails.org/grails/core/org/grails/plugins/hibernate5/)). So it seems it may have been a mistake to depend on `gormVersion` for defining that. I'm guessing that was a relic from when the GORM plugins were organized differently, and shouldn't apply anymore. 

Therefore, I just used the same versions as given in a fresh Grails 4.0 project.

Reply:
For reference, I asked about it here: https://grails.slack.com/archives/C0J9Y5PHU/p1562866992014200

Reply:
@zhuravskiy Thank you for reviewing!